targets/TARGET_ARM_SSG/TARGET_MUSCA_B1/CMakeLists.txt

```diff
@@ -55,3 +55,10 @@ target_link_libraries(mbed-arm-musca-b1
         ${CMAKE_CURRENT_SOURCE_DIR}/s_veneers.o
         mbed-arm-ssg
 )
+
+add_library(mbed-arm-musca-b1-ns INTERFACE)
```

Comment:
The NS target was previous just an alias for the B1 target for backwards compatibility. The Mbed OS B1 target is always NS. The NS alias was removed in PR: https://github.com/ARMmbed/mbed-os/pull/14287

Reply:
Yes, we are merging [feature-tf-m-1.2-integration](https://github.com/ARMmbed/mbed-os/tree/feature-tf-m-1.2-integration) very soon and the NS aliases won't exist anymore

Reply:
OK, I missed this was only merged to the integration branch so far. I'd recommend we merge this PR as-is (because the target still exists on `master`) and make sure when we bring the integration branch in, we ensure we remove this extra target from this CMake file.